br-ext/configs/toolchain-aarch32-sdk

```diff
@@ -0,0 +1,9 @@
+BR2_TOOLCHAIN_EXTERNAL=y
+BR2_TOOLCHAIN_EXTERNAL_CUSTOM=y
+BR2_TOOLCHAIN_EXTERNAL_PATH="%TOP_DIR%/toolchains/aarch32"
+BR2_TOOLCHAIN_EXTERNAL_CUSTOM_PREFIX="arm-linux"
```

Comment:
Why not use the traditional prefix, `arm-linux-gnueabihf`? Wouldn't it simplify things a bit?

Reply:
I don't know how to configure "make sdk" to do that. Here we're using what "make sdk" produced.